src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java

```diff
@@ -0,0 +1,45 @@
+package io.xstefank.wildlfy.bot.format.checks;
+
+import io.xstefank.wildlfy.bot.config.RegexDefinition;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DescriptionCheck implements Check{
+    static final String DEFAULT_MESSAGE = "Invalid description content";
+
+    private Pattern pattern;
+    private String message;
+
+    public DescriptionCheck(RegexDefinition description) {
+        if (description == null || description.pattern == null) {
+            throw new IllegalArgumentException("Input argument cannot be null");
+        }
+        pattern = description.pattern;
+        message = (description.message != null) ? description.message : DEFAULT_MESSAGE;
+
+        System.out.println(pattern);
+        System.out.println(message);
```

Comment:
remove souts

Reply:
either use log or not print

Reply:
I deleted it. In **TitleCheck** there is the same thing